web/packages/teleport/src/Assist/Assist.tsx

```diff
@@ -201,7 +201,7 @@ const Container = styled.div<{ docked: boolean }>`
   opacity: 0;
   animation: forwards ${fadeIn} 0.3s ease-in-out;
   background: rgba(0, 0, 0, 0.5);
-  z-index: 1000;
+  z-index: ${p => (p.docked ? 2 : 100)};
```

Comment:
Long term we should start thinking of using some kind of a map for z-index levels so that we don't get lost in them. I remember briefly talking with Grzegorz about this when we ran into a similar problem in Connect (https://github.com/gravitational/teleport/pull/20988#discussion_r1092179324).

https://css-tricks.com/systems-for-z-index/

Reply:
Agreed. This could also not be positioned absolutely/fixed when docked, but I wanted a quick fix 😅